### ISSUE TITLE
Miniflow - Implementing Backpropagation with Gradient Descent - Answer not checked correctly

### ISSUE THREAD

**Issue by lachezarbozhkov** [udacity/sdc-issue-reports#94]
13. Implementing Backpropagation with Gradient Descent: the answer is incorrect.
In nn.py it says:
"""
You should see a list of gradients on the weights and bias that looks like:
[array([[  6.08973702e-08,   6.93800843e-02],
       [  1.21794740e-07,   1.38760169e-01]]), array([ -6.08973702e-08,  -6.93800843e-02])]
"""

When run with the code provided for sigmoid backward on the next page the result is:
[array([[  6.79488741e-18,  -2.67826355e-12],
       [  1.35897748e-17,  -5.35652711e-12]]), array([ -6.79488741e-18,   2.67826355e-12])]

Totaly different. 
Also whatever the answer when press submit - it says: great, correct answer.

**Comment by georgesung** [comment]
Same observation here.

**Comment by domluna** [comment]
The list of gradients was updated to the correct one:

```
[array([[ 6.79488741e-18, -2.67826355e-12],
[ 1.35897748e-17, -5.35652711e-12]]), array([ -6.79488741e-18, 2.67826355e-12])]
```

There's is a bug in the grader solution which makes it always a successful submission. For now, test the solution by with the expect output.
